p/scripts/main.js

```diff
@@ -720,8 +720,14 @@ function init_shortcuts() {
 				return true;
 			}
 
-			const s = context.shortcuts,
-				k = (ev.key.trim() || ev.code).toUpperCase();
+			const s = context.shortcuts;
+			let k = (ev.key.trim() || ev.code || 'Space').toUpperCase();
+
+			//IE11
```

Comment:
Should probably specify that it's also Edge like the PR title.
```suggestion
			//IE11 / Edge
```

Reply:
Actually, that part is solely IE11. Edge does not specify `ev.code` for a space, which is covered by the previous line, while also preventing a risk of null